### PR TITLE
Add extra contributing steps and move steps to separate file

### PR DESCRIPTION
**Pull request by pffijt**
Added extra steps to contributing a container and moved it to a separate file contributing.md.

### CODE REVIEW

**Comment by pffijt** [comment]
related to #269

**Comment by maltehedderich** [comment]
Hey,

should we also update the pull request template [new_container.md](https://github.com/testcontainers/testcontainers-python/blob/29629e261cbea631f8a4174b8a8a404fa35ee364/.github/PULL_REQUEST_TEMPLATE/new_container.md) with the additions and make sure, that they are consistent?

I like the improvements though 👍🏿

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+### Contributing a new container
+
+You can contribute a new container in three steps:
+
+1. Create a new module with a class for the container at `testcontainers/[my fancy container].py` that implements the new functionality.
+2. Add a working usage example and short functionality description to the new class's docstring.
```
Comment (maltehedderich) [inline contributing.md:8]: Should we add the term 'doctest' here somehow to simplify the research on 'How to do that' for people who have not worked with doctests before? Maybe a reference to one of the existing?

**Comment by tillahoffmann** [comment]
Closing for now (see #269 for details).

**Pull request closed without merging**